plugins/rollbar_dashboards/lib/rollbar_dashboards/client.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module RollbarDashboards
+  class Client
+    def initialize(dashboard)
+      @base = dashboard.base_url
+      @token = dashboard.resolved_read_token
+    end
+
+    def top_errors(hours: 24, environments: ['production'])
+      args = {
+        access_token: @token,
+        hours: hours,
+        environments: environments.join(',')
+      }.to_query
+
+      response = Faraday.get("#{@base}/reports/top_active_items?#{args}")
+      if response.success?
+        data = ::JSON.parse(response.body)
+        data['result'].map { |item_hash| item_hash['item'] }
+      end
+    rescue ::JSON::ParserError
```

Comment:
need to rescue more here ... send to ErrorNotifier and return nil ... we can't blow up peoples project/deploy pages
... and needs a timeout ideally at the http level since `Timeout.timeout` is dangerous